src/pkcs11/pkcs11-object.c

```diff
@@ -693,6 +693,7 @@ C_Sign(CK_SESSION_HANDLE hSession,		/* the session's handle */
 		CK_BYTE_PTR pSignature,		/* receives the signature */
 		CK_ULONG_PTR pulSignatureLen)	/* receives byte count of signature */
 {
+	*pulSignatureLen = 1025;
```

Comment:
```suggestion
```
Let's see if we can live without that now.

Reply:
This can now be removed when using a signature length greater or equal than the length of the signature in sequence for the computation of the signature in card-isoapplet.c